beanquery/query_render.py

```diff
@@ -182,79 +181,49 @@ def format(self, value):
 
 
 class DecimalRenderer(ColumnRenderer):
-    """A renderer for decimal numbers."""
+    """Renderer for Decimal numbers.
+
+    Numbers are left padded to align on the decimal point::
+
+      -  123.40
+      -    5.000
+      -  -67
+
+    """
```

Comment:
@blais For decimals I do only alignment and no quantization or rounding. I think it is best to do not try to be too smart and infer resolution from the distribution of the numers in the query, not only because most probably all numbers are anyhow generated through the same formula and most probably have very similar shape anyhow.